db/migrate/20230315090426_add_scopes_to_token.rb

```diff
@@ -0,0 +1,15 @@
+class AddScopesToToken < ActiveRecord::Migration[7.0]
+  def up
+    add_column :tokens, :scopes, :jsonb, default: [], null: false
+    add_index  :tokens, :scopes, using: :gin
+
+    Token.includes(:scopes).find_each do |token|
```

Comment:
```suggestion
    Token.includes(:old_scopes).find_each do |token|
```

vu que la relation a changé (chelou que ça passe là)